test/views/declaration/LocationViewSpec.scala

```diff
@@ -72,7 +72,7 @@ class LocationViewSpec extends UnitViewSpec with ExportsTestData with Stubs with
 
     "display header" in {
 
-      view.getElementById("title").text() mustBe "declaration.goodsLocation.title"
+      view.getElementsByClass("govuk-label govuk-label--l").text() mustBe "declaration.goodsLocation.title"
     }
```

Comment:
I normally do this to select the title
`view.getElementsByTag("h1").text() mustBe "blah.blah"`
There might be more than one element with those style classes on the page.  (Not in your case because the test would fail as .text() would return all values for all element found (i.e. more than just the title).

So you don't have to change if you don't want to.